src/ontology/edits/oeo-physical.omn

```diff
@@ -269,6 +272,61 @@ Class: <http://openenergy-platform.org/ontology/oeo-physical/ChemicalEnergy>
         Energy
     
     
+Class: <http://openenergy-platform.org/ontology/oeo-physical/PortionOfCarbonDioxide>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Carbon dioxide is a portion of matter with the chemical formula CO2. It is a gas that occurs naturally in the atmosphere as a trace gas and it is a greenhouse gas.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "CO2"
+    
+    SubClassOf: 
+        PortionOfMatter,
+        <http://purl.obolibrary.org/obo/RO_0000091> some GreenhouseGas
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo-physical/PortionOfMethane>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Methane is a portion of matter with the chemical formular CH4. It is a gas that occurs naturally in the atmosphere as a trace gas and it is a greenhouse gas. As it can be oxidised it is a fuel.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "CH4"
+    
+    SubClassOf: 
+        PortionOfMatter,
+        <http://purl.obolibrary.org/obo/RO_0000091> some GreenhouseGas
+    
```

Comment:
Methane is also a fuel, this link is missing.

Reply:
We should discuss #227 first before changing this.

Reply:
It is PortionOfMatter and through equivalent classes now also a PortionOfFuel since I added the disposition Fuel